src/components/VitalStat.vue

```diff
@@ -0,0 +1,127 @@
+<script>
+/**
+ *   Displays a read-only stat
+ * @param stat {JSON} displayed object
+ * @DEPRECATED @param styleProp {String} style of displayed stat points
+ * @param scale {Number} total number of points, split into groups of five
+ * @param value {Number} sum of dependency values
+ */
+export default {
+  props:{'stat': Object, 'scale': Number, 'value': Number},
+  data(){
+    return{
+      initialValue:(this.stat.defaultValue) ? this.stat.defaultValue : this.finalValue()
+    }
+  },
+  methods:{
+    /**
+     * @returns sum of dependecies + modifier
+     */
+    finalValue() {
+      return (this.stat.defaultModifier) ? this.stat.defaultModifier + this.value : this.value;
```

Comment:
ked sa prida defaultModifier=0 k Hunger, tak ta funkcia tu vobec nemusi byt...

Reply:
... hmmm mozno som to zle pochopil :-)

Reply:
tuto sa pocita s tym, ze uz hras hru a nastavujes hunger. Preto mas take data a take funkcie. 

Pre vytlacenie papieroveho deniku mi staci vidiet kolko mam stvorcekov zdravia. Potrebujem ich mat cierne, aby som mohl preskrtavat tuzkou a mazat gumou.

V deniku musi byt viet aj hunger, ale default value 1 tam nemoze byt vytlacena ako preskrtnuta, lebo ju mozem zmazat (stvorcek by tam ani nemal zmysel, keby bol stale preskrtnuty). No ale pretoze jediny hunger ma v datach defaultValue, tak sa moze cely defaultValue dat prec a funkcia pre pocitanie vyslednej hodnoty sa moze zjednodusit